daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/parsers/DelimitedParsers.scala

```diff
@@ -183,3 +185,257 @@ class HexBinaryDelimitedParser(
     }
   }
 }
+
+
+class PackedIntegerDelimitedParser(
+  erd: ElementRuntimeData,
+  textParser: TextDelimitedParserBase,
+  fieldDFAEv: FieldDFAParseEv,
+  isDelimRequired: Boolean,
+  signed: Boolean)
+  extends StringDelimitedParser(erd, TextJustificationType.None, MaybeChar.Nope, textParser, fieldDFAEv, isDelimRequired) {
+
+  /**
+   * HexBinary is just a string in iso-8859-1 encoding.
```

Comment:
Comment is cut/paste from HexBinary. Update or remove, and put in a comment saying this is similar to how hexBinary works.